src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Serialization/Stream.DeserializeAsyncEnumerable.cs

```diff
@@ -82,7 +83,13 @@ public static async Task DeserializeAsyncEnumerable_ShouldStreamPartialData(bool
             string json = JsonSerializer.Serialize(Enumerable.Range(0, 100));
 
             using var stream = new Utf8MemoryStream(json);
-            IAsyncEnumerable<int> asyncEnumerable = DeserializeAsyncEnumerableWrapper<int>(stream, new JsonSerializerOptions { DefaultBufferSize = 1 }, useJsonTypeInfoOverload: useJsonTypeInfoOverload);
+            JsonSerializerOptions options = new JsonSerializerOptions
+            {
+                DefaultBufferSize = 1,
+                TypeInfoResolver = new DefaultJsonTypeInfoResolver()
```

Comment:
These tests are being used both in reflection and source gen contexts. By hardcoding the resolver to the reflection converter this is effectively not testing source generation. It should be reverted.